packages/@aws-cdk/aws-lambda/lib/event-source-mapping.ts

```diff
@@ -126,6 +126,13 @@ export interface EventSourceMappingOptions {
    */
   readonly startingPosition?: StartingPosition;
 
+  /**
+   * The time from which to start reading, in Unix time seconds.
+   *
+   * @default Required when startingPosition is AT_TIMESTAMP.
```

Comment:
same comment regarding this `@default`

Reply:
i also see how `@default` is being used in startingPosition. I consider that wrong also. Would you please change that as well?

Reply:
Updated both